resources/views/_partials/forms/edit-course.blade.php

```diff
@@ -0,0 +1,69 @@
+<form action="{{ route('staff.courses.update', $course) }}" method="POST" 
+    class="space-y-3"
+    x-data="courseForm({
```

Comment:
now this partial depends on a script function which will be added by another function so its better to allow a little duplication and redeclare that single type property here..